src/dlgTriggerEditor.cpp

```diff
@@ -650,15 +517,12 @@ void dlgTriggerEditor::slot_viewStatsAction()
 
 void dlgTriggerEditor::slot_viewErrorsAction()
```

Comment:
Agreed, I saw it as well but could not understand it.  I think I will need to run it through GammaRay and see if I can identify what it is, probably a widget that should be hidden somewhere... :confused: